src/uct/ugni/smsg/ugni_smsg_iface.c

```diff
@@ -208,6 +209,7 @@ static ucs_status_t uct_ugni_smsg_iface_query(uct_iface_h tl_iface, uct_iface_at
     memset(iface_attr, 0, sizeof(uct_iface_attr_t));
     iface_attr->cap.am.max_short       = iface->config.smsg_seg_size-sizeof(uint64_t);
     iface_attr->cap.am.max_bcopy       = iface->config.smsg_seg_size;
+    iface_attr->device_addr_len        = sizeof(uct_devaddr_ugni_t);
```

Comment:
You can, I tested it and it makes no difference in size. I'll set it to 0 anyways since you have to connect to eps anyways and not iface.